src/index.html

```diff
@@ -6,13 +6,62 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="./main.js"
+    />
```

Comment:
The link to the JavaScript file is incorrect. You are using the 'rel="stylesheet"' attribute which is for CSS files. For JavaScript files, you should use the 'script' tag with 'src' attribute to link the JavaScript file.